README.md

```diff
@@ -38,6 +38,8 @@ Questions are grouped into three buckets: [General](#general), [Technical](#tech
     + What kind of elements can `title` attributes be used on?
     + What sort of information is appropriate for use with the `title` attribute?
 - Describe a scenario where you might need to use `aria-describedby`.
+- What is "tabtrapping" and when should it be used?
```

Comment:
suggest rewording to:
"What is a focus trap, or focus trapping?  Describe an instance of when you'd need focus trapping, and how it can be an accessibility failure."

Reply:
awesome; updated